lib/src/fields/form_builder_range_slider.dart

```diff
@@ -170,24 +178,27 @@ class FormBuilderRangeSlider extends FormBuilderField<RangeValues> {
                     children: <Widget>[
                       if (displayValues != DisplayValues.none &&
                           displayValues != DisplayValues.current)
-                        Text(
-                          effectiveNumberFormat.format(min),
-                          style: minTextStyle ?? textStyle,
-                        ),
+                        minValueWidget ??
```

Comment:
I think that all new properties will be a function like `Widget Function(String)? minValueWidget` and here apply `minValueWidget?.call(effectiveNumberFormat.format(min))`.
With this way can access to format values in external widget. What do you think?

Reply:
I'm a little confused about what you mean.

Let's say the max value is 100, then the number under the right side of the slider will say 100.

I want to change that text widget to be something other than a text widget saying "100".

I want it to be an Icon widget.

Reply:
Aaah ok, I understand what you mean. I thought that you will be show the `min` value inside the widget.
Either way, even you don't use the min/max value, will be nice have this value available to build the custom widget.
For example: you can create a Text with icon and the text value will be a min/max, like this:

```dart
min: 100
minValueWidget: (min) => Row(children:[Text(min), Icon(Icons.max)]) /// This min will be 100
```
With this way you can reuse the min value without duplicate again

Reply:
That sounds good! Let me know if there's anything else!